src/core_atmosphere/Registry.xml

```diff
@@ -1135,6 +1135,72 @@
                         <var_array name="lbc_scalars" />
 
                 </stream>
+		<stream name="da_state"
```

Comment:
Could you add a blank line between the end of the "lbc_in" stream and the beginning of the "da_state" stream?

Reply:
Yes.

Reply:
Done.